app/routes/offender-manager-utilisation.js

```diff
@@ -0,0 +1,11 @@
+const ViewUtilisation = require('../services/domain/view-utilisation')
+
+module.exports = function (router) {
+  router.get('/offender-manager-utilisation/:userId/:year/', function (req, res, next) {
```

Comment:
User Id implies it's a user of the system which won't always be the case. `offenderManagerId` would probably be more accurate and reflective of how it'll be named in the DB